docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -293,21 +293,21 @@ The following command runs tests and specifies a custom reporter plugin that gen
 testcafe all tests/sample-fixture.js -r my-reporter
 ```
 
-The generated test report is displayed in the command prompt window.
+The command prompt window displays the test report generated.
 
-If you need to save the report to an external file, specify the file path after the report name.
+To save the report to an external file, specify the file path after the report name.
 
 ```sh
 testcafe all tests/sample-fixture.js -r json:report.json
 ```
 
-You can also use multiple reporters in a single test run. List them separated by commas.
+You can use multiple reporters in a single test run. List them separated by commas.
 
 ```sh
 testcafe all tests/sample-fixture.js -r spec,xunit:report.xml
 ```
 
-Note that only one reporter can write to `stdout`. All other reporters must output to files.
+Note that one reporter can write to `stdout` at one time. All other reporters must output to files.
```

Comment:
Or `at a time`?

Or better... We are not talking about writing simultaneously, we just mean "during the same test run".

`during the same run`?

Reply:
> During a test, only one reporter can write to `stdout`. Redirect other reporters' output to files.